challenges/objects-arrays.js

```diff
@@ -98,6 +139,9 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 */
 
 const lowerCase = [];
+zooAnimals.map((animal) => {
+  lowerCase.push(animal.animal_name.toLowerCase());
```

Comment:
You have to remember when using .map() not to use push since .map() already returns a new array. Great work on getting this to work but just unnecessary using .push().